net-im/telegram-desktop/telegram-desktop-5.12.3-r1.ebuild

```diff
@@ -41,28 +41,13 @@ CDEPEND="
 	enchant? ( app-text/enchant:= )
 	jemalloc? ( dev-libs/jemalloc:= )
 	libdispatch? ( dev-libs/libdispatch )
-	!qt6? (
-		>=dev-qt/qtcore-5.15:5=
-		>=dev-qt/qtgui-5.15:5=[dbus?,jpeg,png,wayland?,X?]
-		>=dev-qt/qtimageformats-5.15:5
-		>=dev-qt/qtnetwork-5.15:5[ssl]
-		>=dev-qt/qtsvg-5.15:5
-		>=dev-qt/qtwidgets-5.15:5[png,X?]
-		kde-frameworks/kcoreaddons:5
-		webkit? ( wayland? (
-			>=dev-qt/qtdeclarative-5.15:5
-			>=dev-qt/qtwayland-5.15:5[compositor(+)]
-		) )
-	)
-	qt6? (
-		>=dev-qt/qtbase-6.5:6=[dbus?,gui,network,opengl,wayland?,widgets,X?]
-		>=dev-qt/qtimageformats-6.5:6
-		>=dev-qt/qtsvg-6.5:6
-		webkit? ( wayland? (
-			>=dev-qt/qtdeclarative-6.5:6
-			>=dev-qt/qtwayland-6.5:6[compositor,qml]
-		) )
-	)
+	>=dev-qt/qtbase-6.5:6=[dbus?,gui,network,opengl,wayland?,widgets,X?]
+	>=dev-qt/qtimageformats-6.5:6
+	>=dev-qt/qtsvg-6.5:6
+	webkit? ( wayland? (
+		>=dev-qt/qtdeclarative-6.5:6
+		>=dev-qt/qtwayland-6.5:6[compositor,qml]
+	) )
```

Comment:
Please move these dependencies to sit above the use-flag dependencies. (unless there's a gentoo style guide mandating otherwise? idk I've never read it lol)

Reply:
Wait, don't move the webkit portion, just the ones that aren't use-conditional anymore.